ingest/validation/validate_metadata.py

```diff
@@ -633,9 +633,7 @@ def validate_input_metadata(metadata, convention, bq_json=None):
     with open(args.convention, 'r') as f:
         convention = json.load(f)
     filetsv = args.input_metadata
-    metadata = CellMetadata(
-        filetsv, args.file_id, args.study_accession, open_as='dataframe'
-    )
+    metadata = CellMetadata(filetsv, args.file_id, args.study_accession)
```

Comment:
So we've taken out `study_accession` in favor of `study_id` (because that's what MongoDB wants) but in the case of cell metadata we probably still need it.  So that may need to get added back into the command line for `ingest_pipeline.py`.

Reply:
Addressed in d01e080